worlds/Arena.wbt

```diff
@@ -41,6 +41,13 @@ Robot {
   description "A supervisor 'robot' to help run the competition."
   controller "competition_supervisor"
   supervisor TRUE
+  children [
+    # this is a hack to detect if we are running on version 2020b
```

Comment:
So what's the idea here? Have a useless generic device which can be retrieved by `getDevice`, simply to check it returns null?

Could this name be added to the supervisor itself? Save adding an extra node?

Reply:
I did try that but in order for `getDevice` to return something on 2021a you need a node that subclasses `Device` that is a child of the robot the controller is running on.